specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/examples/RedisCacheCreate.json

```diff
@@ -6,6 +6,7 @@
     "subscriptionId": "subid",
     "parameters": {
       "location": "West US",
+      "redisVersion": "4",
```

Comment:
this looks like its at the wrong level, it should be inside 'properties' right?

Reply:
Fixed it. Thanks.